src/popup/router/components/Modals/QrCodeReader.vue

```diff
@@ -190,5 +205,28 @@ export default {
       max-width: 100%;
     }
   }
+
+  .subtitle {
+    @extend %face-sans-14-bold;
+
+    color: variables.$color-light-grey;
+  }
+
+  .text {
+    @extend %face-sans-13-medium;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
 }
+
+.modal .container .footer .content .button {
+  width: 100%;
+}
+
+.modal::v-deep .container .body {
```

Comment:
I strongly advise agains  overriding child's component's styles within the parent. Here you are not able to be 100% sure what CSS class names will be used in the child component in the future. So this makes the code harder to maintain.

Reply:
It needs to wait for 'Receive popup' epic then